lib/commands/deprecated/activate.js

```diff
@@ -0,0 +1,31 @@
+var chalk = require('chalk');
+var Promise = require('ember-cli/lib/ext/promise');
+var SilentError         = require('ember-cli/lib/errors/silent');
+var DeprecatedCommand = require('./deprecated-command');
+var ActivateCommand = require('../activate');
+
+module.exports = DeprecatedCommand({
+  name: 'activate',
+  description: 'Activate a deployed version of index.html',
+  works: 'insideProject',
+
+  anonymousOptions: [
+    '<key>'
+  ],
+
+  availableOptions: [
+    { name: 'environment', type: String, default: 'development' },
+    { name: 'deploy-config-file', type: String, default: 'config/deploy.js' }
```

Comment:
@achambers - I used the new location of config files, I am leaving the notification to users of the original `ember-cli-deploy` package to a later PR.